src/app/preferences/_preferences.html

```diff
@@ -8,7 +8,7 @@
 		{{'PREFERENCES_MANAGE_SETTINGS_PREFERENCES' | translate}}<br />
 	</p>
 	
-    <div class="menu">
+    <!--<div class="menu">
```

Comment:
Indeed, if I go to preferences page, I don't want to display first the form to choose language without click on "Languages".